src/documentation/pages/components/button/examples/_floating-icon-lumen-centurylink.vue

```diff
@@ -0,0 +1,57 @@
+<template lang="pug">
+  <ComponentExample title="Floating icon buttons" id="floating-icon-lumen-centurylink" :tabs="exampleTabs">
+    div(slot="example")
+      .-d--flex
```

Comment:
does .-d--flex(slot="example") work? (without a div above)

Reply:
Good catch. Yes, it should work.